src/layer/DivOverlay.js

```diff
@@ -159,38 +159,45 @@ export var DivOverlay = Layer.extend({
 		return this;
 	},
 
-	_prepareOpen: function (parent, layer, latlng) {
-		if (!(layer instanceof Layer)) {
-			latlng = layer;
-			layer = parent;
-		}
-
-		if (layer instanceof FeatureGroup) {
-			for (var id in parent._layers) {
-				layer = parent._layers[id];
-				break;
+	// prepare bound overlay to open: update latlng pos / content source (for FeatureGroup)
+	_prepareOpen: function (latlng) {
+		var source = this._source;
+		if (!source._map) { return; }
+
+		if (source instanceof FeatureGroup) {
+			source = null;
+			var layers = this._source._layers;
+			for (var id in layers) {
+				if (layers[id]._map) {
+					source = layers[id];
+					break;
+				}
 			}
+			if (!source) { return; } // Unable to get source layer.
+
+			// set overlay source to this layer
+			this._source = source;
 		}
 
 		if (!latlng) {
-			if (layer.getCenter) {
-				latlng = layer.getCenter();
-			} else if (layer.getLatLng) {
-				latlng = layer.getLatLng();
-			} else if (layer.getBounds) {
-				latlng = layer.getBounds().getCenter();
+			if (source.getCenter) {
+				latlng = source.getCenter();
+			} else if (source.getLatLng) {
+				latlng = source.getLatLng();
+			} else if (source.getBounds) {
+				latlng = source.getBounds().getCenter();
 			} else {
 				throw new Error('Unable to get source layer LatLng.');
 			}
 		}
+		this.setLatLng(latlng);

```

Comment:
Explain please

Reply:
`setLatLng` calls `_adjustPan` and in the Popup a event is fired: 
https://github.com/Leaflet/Leaflet/blob/f018e2de2ce507ab9b070f382e6bded9b9311263/src/layer/Popup.js#L273-L277

But I don't think that this will make problems